service/project.go

```diff
@@ -547,6 +548,15 @@ func (uis *UIServer) setRevision(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// run the repotracker for the project
+	ts := util.RoundPartOfHour(0).Format("2006-01-02.15-04-05")
+	j := units.NewRepotrackerJob(fmt.Sprintf("catchup-set-revision-%s", ts), projectRef.Identifier)
```

Comment:
Do we want to use the same identifier pattern as the regular catchup job so that we don't run 2 of them at the same time?

Reply:
doen

Reply:
done

Reply:
I think you need to use a similar time rounding format as the job for this to work. The existing job rounds to 30 mins, but this one can probably round to 5?

Reply:
done